lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -39,12 +42,60 @@ defmodule MembraneLiveWeb.EventChannel do
     {:error, %{reason: "This link is wrong."}}
   end
 
-  def handle_info({:after_join, name}, socket) do
+  def join_event_stream(event_id, socket) do
+    case :global.whereis_name(event_id) do
+      :undefined -> Event.start(event_id, name: {:global, event_id})
+      pid -> {:ok, pid}
+    end
+    |> case do
+      {:ok, event_pid} ->
+        do_join(socket, event_pid, event_id)
+
+      {:error, {:already_started, event_pid}} ->
+        do_join(socket, event_pid, event_id)
+
+      {:error, reason} ->
+        Logger.error("""
+        Failed to start room.
+        Room: #{inspect(event_id)}
+        Reason: #{inspect(reason)}
+        """)
+
+        {:error, %{reason: "failed to start room"}}
```

Comment:
As we don't define the name Room in this context, I suggest changing Room to Event or Event stream.
```suggestion
        {:error, %{reason: "failed to start event"}}
```